packages/rangelink-core-ts/src/__tests__/matchers/toBeRangeLinkError.test.ts

```
@@ -0,0 +1,195 @@
+import { RangeLinkError } from '../../errors/RangeLinkError';
+import { RangeLinkErrorCodes } from '../../errors/RangeLinkErrorCodes';
+import { Result } from '../../types/Result';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)